src/main/java/com/example/booking_app/repository/ConfigurationRepository.java

```diff
@@ -0,0 +1,136 @@
+package com.example.booking_app.repository;
+
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+@Repository
+public class ConfigurationRepository  {
+
+    @Autowired
+    CinemaSeatRepository cinemaSeatRepository;
+
+    @Autowired
+    ReservationRepository reservationRepository;
+
+    @Autowired
+    ScreeningRepository screeningRepository;
+
+    private final JdbcTemplate jdbcTemplate;
+
+    @Autowired
```

Comment:
Czemu część wstrzykujesz konstruktorem a część polem?

Reply:
Jest to mój błąd. Adnotacja @Autowired powinna znajdować się nad polem jdbcTemplate tak jak jest w pozostałych klasach.